homeassistant/components/withings/coordinator.py

```diff
@@ -66,40 +67,62 @@
     NotifyAppli.BED_IN: Measurement.IN_BED,
 }
 
+_T = TypeVar("_T")
+
 UPDATE_INTERVAL = timedelta(minutes=10)
 
 
-class WithingsDataUpdateCoordinator(DataUpdateCoordinator[dict[Measurement, Any]]):
+class WithingsDataUpdateCoordinator(DataUpdateCoordinator[_T]):
     """Base coordinator."""
 
-    in_bed: bool | None = None
     config_entry: ConfigEntry
+    _default_update_interval: timedelta | None = UPDATE_INTERVAL
+    notification_categories: list[NotifyAppli] = []
```

Comment:
Looks like this should be a set instead.

Also, please avoid initializing mutable objects as class attributes.